app/initializers/document-source.js

```diff
@@ -14,5 +14,7 @@ export default {
     application.inject('route', 'documentSource', 'document-source:main');
     application.inject('controller', 'documentSource', 'document-source:main');
     application.inject('adapter:application', 'documentSource', 'document-source:main');
+    application.inject('adapter:google-drive-permission', 'documentSource', 'document-source:main');
+    application.inject('component:document-creator', 'documentSource', 'document-source:main');
   }
 };
```

Comment:
I've noticed there were a couple of initializers in charge of injecting `documentSource` into various parts of the application. I decided to move them all into a single initializer, since they're supposed to run at the same time. I find this easier to manage.